resources/test/tests/idlharness-primary-int-obj.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>idlharness: Primary interface</title>
+  <script src="../../testharness.js"></script>
+  <script src="../../testharnessreport.js"></script>
+  <script src="../../WebIDLParser.js"></script>
+  <script src="../../idlharness.js"></script>
+</head>
+<body>
+<p>Regression test for prior bug where tests were skipped for "tested"
```

Comment:
As discussed on irc, I think that's more useful in the versioning system than here.